lib/compat/wordpress-6.5/block-bindings/sources/post-meta.php

```diff
@@ -17,6 +17,13 @@ function gutenberg_block_bindings_post_meta_callback( $source_attrs ) {
 		$post_id = get_the_ID();
 	}
 
+	// If a post isn't public, we need to prevent
+	// unauthorized users from accessing the post meta.
+	$post = get_post( $post_id );
+	if ( ( $post && 'publish' !== $post->post_status && ! current_user_can( 'read_post', $post_id ) ) || post_password_required( $post_id ) ) {
+		return null;
+	}
+
```

Comment:
Did you mean to include this change as well?

I think this was pointed out to us in some comment (I can't find it now) but should probably be in a separate PR.

Reply:
If you meant to include it let's just note it in the PR description 🙂

Reply:
It was included in the WordPress core backport: [link](https://github.com/WordPress/wordpress-develop/pull/5888/files#r1467885715).

I added it here because it seemed approved there, trying to avoid too many pull requests that need the "Backport to RC". But I can move it to another pull request if you consider it better.